.github/workflows/codeql-analysis.yml

```diff
@@ -33,7 +33,7 @@ jobs:
 
       # Initializes the CodeQL tools for scanning.
       - name: Initialize CodeQL
-        uses: github/codeql-action/init@2ca79b6fa8d3ec278944088b4aa5f46912db5d63
+        uses: github/codeql-action/init@f5d217be74900c6ac8fbbe53f3c10376ba4e64da
```

Comment:
I'm curious why this change is included in this PR. https://github.com/dart-lang/site-www/pull/4180 landed this change already, IIUC.

@parlough any ideas?

Reply:
I assume you noticed, but this is a separate branch tracking 2.18.

Reply:
> I assume you noticed, but this is a separate branch tracking 2.18. This is just merging those new commits in.

Yeah i figured it out finally =)

Reply:
Ahh, sorry all, I was refreshing myself on git. You can ignore this for the most part, it's just bringing this branch up to speed with main (my commits will be moved to a different pr)